### PR TITLE
Fix broken demo and security links in README.md

### PR DESCRIPTION
**Pull request by zsais**
Repair broken links in README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Learn more in [Getting Started](https://google.github.io/mesop/getting-started/i
 
 ## Reporting a security issue
 
-If you identify a security vulnerability with Mesop, please file a [GitHub security issue](https://github.com/google/mesop/security/advisories/new) and we will promptly respond to it.
+If you identify a security vulnerability with Mesop, please file a [GitHub security issue](https://github.com/google/mesop/security/) and we will promptly respond to it.
```
Comment (richard-to) [inline README.md:84]: Maybe update to this link here: https://github.com/google/mesop/security/policy

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Mesop is a Python-based UI framework that allows you to rapidly build web apps l
 
 ## Write your first Mesop app in less than 10 lines of code...
 
-[Demo app](https://google.github.io/mesop/demo/?demo=text_io)
+[Demo app](https://google.github.io/mesop/demo/)
```
[the remark this answers is not in the document]
Comment (zsais) [reply README.md:33]: Okay I'll point the deep link at that other demo. Should we update the sample code in this as well?

[the remark this answers is not in the document]
Comment (richard-to) [reply README.md:33]: Sure. I think we can update the sample code to match what the current demo is.

I think to keep the sample code as simple as possible, we can remove the stuff that sets the system theme and the security policy.

**Review by richard-to — APPROVED** [review]
LGTM - Thanks for the edits, Zach.

**Pull request merged**